test-cases/README.md

```diff
@@ -0,0 +1,183 @@
+# Integreatly Test Cases
+
+- Test cases are located in the directory `/tests` and are organized in subdirectories by categories.
```

Comment:
```suggestion
 - Test cases are located in the directory `/test-cases` and are organized in subdirectories by categories. 
```

Reply:
Need to update any other references also, fyi

Reply:
The README is located in the `/test-cases` directory but the tests (.md files) are in the `test-cases/tests` directory, so just `tests/`, but I should remove the `/`